<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>original_map</class>
 <widget class="QDialog" name="original_map">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2342</width>
    <height>1831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="mappic">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>1491</width>
     <height>1611</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/kenzy/Dropbox/My PC (DESKTOP-R153SN6)/Downloads/Map finished.PNG</pixmap>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>-130</x>
     <y>-490</y>
     <width>2641</width>
     <height>2501</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="location_input">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>130</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="destination_input">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>230</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1600</x>
     <y>130</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>From:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1600</x>
     <y>240</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>To:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirming_trip">
   <property name="geometry">
    <rect>
     <x>2060</x>
     <y>530</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>confirm</string>
   </property>
  </widget>
  <widget class="QComboBox" name="car_choice">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>340</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Taxi</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ordinary Car</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Luxurious Car</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1740</x>
     <y>1040</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Mistral</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Driver:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1650</x>
     <y>1120</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Name: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>1200</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Age: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>1340</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Car: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>1420</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>CarPlate: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>1270</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Driver's Rate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>1480</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Car Rate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="driver_name">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>1130</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="age_driver">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>1220</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="driver_rate">
   <property name="geometry">
    <rect>
     <x>1850</x>
     <y>1280</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="car_type">
   <property name="geometry">
    <rect>
     <x>1740</x>
     <y>1350</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="carplate">
   <property name="geometry">
    <rect>
     <x>1790</x>
     <y>1430</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="car_rate">
   <property name="geometry">
    <rect>
     <x>1790</x>
     <y>1500</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>1650</x>
     <y>1660</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Trip price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="price_trip">
   <property name="geometry">
    <rect>
     <x>1780</x>
     <y>1670</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>1650</x>
     <y>1560</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Location:</string>
   </property>
  </widget>
  <widget class="QLabel" name="car_location">
   <property name="geometry">
    <rect>
     <x>1800</x>
     <y>1590</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>1850</x>
     <y>1670</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>LE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>2020</x>
     <y>1570</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Distance:</string>
   </property>
  </widget>
  <widget class="QLabel" name="distance_label">
   <property name="geometry">
    <rect>
     <x>2176</x>
     <y>1573</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>2230</x>
     <y>1580</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>km</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_trip">
   <property name="geometry">
    <rect>
     <x>2010</x>
     <y>1710</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm trip</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>2020</x>
     <y>1490</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>2220</x>
     <y>1490</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>minutes</string>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>2130</x>
     <y>1480</y>
     <width>47</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>mappic</zorder>
  <zorder>location_input</zorder>
  <zorder>destination_input</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>confirming_trip</zorder>
  <zorder>car_choice</zorder>
  <zorder>label</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>driver_name</zorder>
  <zorder>age_driver</zorder>
  <zorder>driver_rate</zorder>
  <zorder>car_type</zorder>
  <zorder>carplate</zorder>
  <zorder>car_rate</zorder>
  <zorder>label_10</zorder>
  <zorder>price_trip</zorder>
  <zorder>label_11</zorder>
  <zorder>car_location</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>distance_label</zorder>
  <zorder>label_15</zorder>
  <zorder>confirm_trip</zorder>
  <zorder>label_14</zorder>
  <zorder>label_16</zorder>
  <zorder>time</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
